core/src/Dscp/Core/Genesis.hs

```diff
@@ -116,7 +116,8 @@ formGenesisInfo GenesisConfig{..} =
                           , txOuts = txOutputs }
                 twWitness = TxWitness { txwSig = sign sk (toTxId twTx, pk, ())
                                       , txwPk = pk }
-            in GMoneyTxWitnessed $ TxWitnessed {..}
+            -- Zero here means minimal fee, since its a genesis.
+            in carryingNothing $ GMoneyTxWitnessed TxWitnessed {..}
```

Comment:
By the way, the genesis block is applied without checks, so tx validator won't kick in anyway.